src/App.jsx

```diff
@@ -14,45 +16,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_BY_NAME = 'Sort alphabetically';
+const SORT_BY_LENGTH = 'Sort by length';
+
+const buttons = [
+  { sortType: SORT_BY_NAME, buttonStyle: 'is-info' },
+  { sortType: SORT_BY_LENGTH, buttonStyle: 'is-success' },
+  { sortType: 'Reverse', buttonStyle: 'is-warning' },
+];
+
+const sortByParams = (toSortArray, sortType, reversing) => {
+  const array = [...toSortArray];
+
+  switch (sortType) {
+    case SORT_BY_NAME:
+      array.sort((a, b) => a.localeCompare(b));
+      break;
+    case SORT_BY_LENGTH:
+      array.sort((a, b) => a.length - b.length);
+      break;
+    default:
+      break;
+  }
+
+  return reversing ? array.reverse() : array;
+};
 
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+export const App = () => {
+  const [sortType, setType] = useState('');
+  const [reversed, setReverse] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig